pkg/csi/cinder/openstack/openstack_snapshots.go

```diff
@@ -37,9 +37,10 @@ const (
 	snapReadyFactor     = 1.2
 	snapReadySteps      = 10
 
-	snapshotDescription = "Created by OpenStack Cinder CSI driver"
-	SnapshotForceCreate = "force-create"
-	SnapshotType        = "type"
+	snapshotDescription      = "Created by OpenStack Cinder CSI driver"
+	SnapshotForceCreate      = "force-create"
+	SnapshotType             = "type"
+	SnapshotAvailabilityZone = "availability"
```

Comment:
I think we need to add this new one to the docs [1]? Also "availability" is ambiguous and might suggest some true-false value. Why not "availability-zone"?

[1] https://github.com/kubernetes/cloud-provider-openstack/blob/e87f50614b96a6af6095660aa6a77457223b344b/docs/cinder-csi-plugin/using-cinder-csi-plugin.md#L263-L273

Reply:
other csi things are using `availability` already. So I thought that I am not going to change that naming. Yep, I need add that to documentation

Reply:
Alright, I'm for consistency with the other CSIs then. Please just add this to docs and I think this is good to go.

Reply:
@dulek done